RecoHI/HiTracking/python/HiTracking_cff.py

```diff
@@ -25,62 +25,69 @@
 from RecoJets.JetAssociationProducers.trackExtrapolator_cfi import *
 trackExtrapolator.trackSrc = cms.InputTag("hiGeneralTracks")
 
-hiTracking_noRegitMu = cms.Sequence(
-    hiBasicTracking
-    *hiDetachedTripletStep
-    *hiLowPtTripletStep
-    *hiPixelPairStep
+hiTracking_noRegitMuTask = cms.Task(
+    hiBasicTrackingTask
+    ,hiDetachedTripletStepTask
+    ,hiLowPtTripletStepTask
+    ,hiPixelPairStepTask
     )
+hiTracking_noRegitMu = cms.Sequence(hiTracking_noRegitMuTask)
 
-hiTracking_noRegitMu_wSplitting = cms.Sequence(
-    hiInitialJetCoreClusterSplitting
-    *hiBasicTracking
-    *hiDetachedTripletStep
-    *hiLowPtTripletStep
-    *hiPixelPairStep
+hiTracking_noRegitMu_wSplittingTask = cms.Task(
+    hiInitialJetCoreClusterSplittingTask
+    ,hiBasicTrackingTask
+    ,hiDetachedTripletStepTask
+    ,hiLowPtTripletStepTask
+    ,hiPixelPairStepTask
     )
+hiTracking_noRegitMu_wSplitting = cms.Sequence(hiTracking_noRegitMu_wSplittingTask)
 
-hiTracking_noRegitMu_wSplitting_Phase1 = cms.Sequence(
-    hiInitialJetCoreClusterSplitting
-    *hiBasicTracking
-    *hiLowPtQuadStep#New iteration
-    *hiHighPtTripletStep#New iteration
-    *hiDetachedQuadStep#New iteration
-    *hiDetachedTripletStep
-    *hiLowPtTripletStep
-    *hiPixelPairStep #no CA seeding implemented
-    *hiMixedTripletStep # large impact parameter tracks
-    *hiPixelLessStep    # large impact parameter tracks
-    *hiTobTecStep       # large impact parameter tracks
+hiTracking_noRegitMu_wSplitting_Phase1Task = cms.Task(
+    hiInitialJetCoreClusterSplittingTask
+    ,hiBasicTrackingTask
+    ,hiLowPtQuadStepTask#New iteration
+    ,hiHighPtTripletStepTask#New iteration
+    ,hiDetachedQuadStepTask#New iteration
+    ,hiDetachedTripletStepTask
+    ,hiLowPtTripletStepTask
+    ,hiPixelPairStepTask #no CA seeding implemented
+    ,hiMixedTripletStepTask # large impact parameter tracks
+    ,hiPixelLessStepTask    # large impact parameter tracks
+    ,hiTobTecStepTask       # large impact parameter tracks
     )
+hiTracking_noRegitMu_wSplitting_Phase1 = cms.Sequence(hiTracking_noRegitMu_wSplitting_Phase1Task)
```

Comment:
This was only needed here internally: sequence not needed any more now, please remove